frontend/src/components/MainSearch/MainSearch.js

```diff
@@ -49,18 +55,43 @@ class MainSearch extends Component {
             </p>
             <span className="search">
               <form className="search-form" onSubmit={this.handleSubmit}>
-                <input
-                  type="text"
-                  className="search-input"
-                  placeholder="Search by keywords"
-                  onChange={this.handleChange}
-                />
-                <button type="submit" className="search-button">
-                  <img
-                    src="https://img.icons8.com/ios/30/000000/search.png"
-                    alt="search"
-                  />
-                </button>
+                <div className="absoluteContainer">
+                  <div className="searchContainer">
+                    <input
+                      type="text"
+                      className="search-input"
+                      placeholder="Search by recipe name or ingredient"
+                      onChange={this.handleChange}
+                    />
+                    <button type="submit" className="search-button">
+                      <img
+                        src="https://img.icons8.com/ios/30/000000/search.png"
+                        alt="search"
+                      />
+                    </button>
+                  </div>
+                  <div className="buttonContainer">
+                    {!this.state.open ? (
+                      <div onClick={(event) => this.togglePanel(event)} className="button-box">
+                        <img
+                          src="https://img.icons8.com/ios/26/000000/chevron-down.png"
+                          alt="chevron button"
+                          className="chevron"
+                        />
+                      </div>
+                    ) : null}
+                  </div>
+                  {/* ternary operator shortcut (if/else) */}
+                  {this.state.open ? (
```

Comment:
I imagine this is causing a linter error. You'll want to do something like `const { open } = this.state` at the top of the function, and use `open` instead of accessing state directly here.

Reply:
So if I change togglepanel to:

togglePanel() {
     const { open } = this.state;
}

I can use:

{this.open ? (div) : null}

for the ternary statements?